src/content/files/cosi-install.sh

```diff
@@ -939,13 +939,18 @@ cosi_register() {
         return
     fi
 
-    log "Running COSI registration script"
-
     [[ -x "$cosi_script" ]] || fail "Unable to find cosi command '${cosi_script}'"
 
+    echo
```

Comment:
This helps to identify messages that are emitted by cosi-install.sh as opposed to (/setup/index.js, etc.) in the console.

The use of echo is precedented a few lines above:
https://github.com/HeinrichHartmann/circonus-one-step-install/blob/0089e9bfdbd9a17ff2446d4d6a55bae8ab1d0093/src/content/files/cosi-install.sh#L931